packages/core/core/src/ReporterRunner.js

```diff
@@ -49,7 +49,19 @@ export default class ReporterRunner {
     let reporters = await this.config.getReporters();
 
     for (let reporter of reporters) {
-      await reporter.report(event, this.pluginOptions);
+      try {
+        await reporter.plugin.report({
+          event,
+          options: this.pluginOptions,
+          // Should this even have a logger? Might end up in an infinite loop...
+          logger: new PluginLogger({origin: reporter.name})
```

Comment:
Probably - some reporters won’t be loggers, eg the dev server, bundle visualizers, etc. They might want to log stuff to different reporters.